Automattic/newspack-plugin
# feat: (IA) dashboard sections

### All Submissions:


### Changes proposed in this Pull Request:

Adds sections with; title, description and group of cards to new dashboard. See below:

### How to test the changes in this Pull Request:

- Checkout this branch and build assets i.e. `git checkout feat/ia-dash-sections && npm run build`
- Navigate to - /wp-admin/admin.php?page=newspack#/
- Observe the following sections exist:

**Audience Development:**

![Screenshot 2024-03-25 at 19 05 25](https://github.com/Automattic/newspack-plugin/assets/3676975/28327137-d962-4377-903a-adfa6a49fcff)

**Newsletters:**

![Screenshot 2024-03-25 at 19 06 28](https://github.com/Automattic/newspack-plugin/assets/3676975/a6328678-02ba-416a-9ead-709d200d5044)

**Advertising:**

![Screenshot 2024-03-25 at 19 07 06](https://github.com/Automattic/newspack-plugin/assets/3676975/ce94ca93-4460-40b6-9e98-a21f065326bd)

**Listings:**

![Screenshot 2024-03-25 at 19 07 41](https://github.com/Automattic/newspack-plugin/assets/3676975/516b24f4-cad0-43f7-bfcb-a7785b134f6a)

**Network:**

![Screenshot 2024-03-25 at 19 07 57](https://github.com/Automattic/newspack-plugin/assets/3676975/637a91a5-1b65-4971-bc86-96818bc7c9d0)

### Other information:
